tiatoolbox/models/engine/patch_predictor.py

```diff
@@ -44,6 +44,90 @@ def __init__(
 class PatchPredictor:
     """Patch-level predictor.
 
+    The models provided by tiatoolbox should give the following results:
+
+    .. list-table:: PatchPredictor performance on the Kather100K dataset [1]
+       :widths: 15 15
+       :header-rows: 1
+
+       * - Model name
+         - F$_1$ score
```

Comment:
```suggestion
         - F_1 score
```

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/335/commits/ad144172ff82bc40d3dceca5fea8c7adadd7eb48